src/SprykerEco/Shared/Amazonpay/AmazonpayConfig.php

```diff
@@ -118,4 +118,36 @@ public function getWidgetScriptPathSandbox()
         return $this->get(AmazonpayConstants::WIDGET_SCRIPT_PATH_SANDBOX);
     }
 
+    /**
+     * @return bool
+     */
+    public function getPopupLogin()
+    {
+        return (bool) $this->get(AmazonpayConstants::WIDGET_POPUP_LOGIN);
```

Comment:
The CS checker will not like the space after casts.

Reply:
Running CS now :)
thanks!